modules/nf-core/deepvariant/callvariants/main.nf

```diff
@@ -0,0 +1,61 @@
+
+process DEEPVARIANT_CALLVARIANTS {
+    tag "$meta.id"
+    label 'process_high'
+
+    //Conda is not supported at the moment
+    container "nf-core/deepvariant:1.5.0"
+
+    input:
+    tuple val(make_examples_id), val(meta), path(make_examples_tfrecords)
+    val model_type
+
+    output:
+    tuple val(make_examples_id), val(meta), path("${prefix}.call.tfrecord.gz"),     emit: call_variants_tfrecords
+    path "versions.yml",                                                            emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        error "DEEPVARIANT module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+
+    def matcher = make_examples_tfrecords[0].baseName =~ /^(.+)-\d{5}-of-(\d{5})$/
+    if (!matcher.matches()) {
+        throw new IllegalArgumentException("tfrecord baseName '" + make_examples_tfrecords[0].baseName + "' doesn't match the expected pattern")
+    }
+    def examples_tfrecord_name = matcher[0][1]
+    def shardCount = matcher[0][2]
```

Comment:
Is it ok that this is e.g. 00003 and not 3?

Reply:
This works in practice, so I think it should be safe. (Some systems use prefix "0" to mean "octal" but this is confirmed to work with >= 8 shards.)